presto-main/src/main/java/com/facebook/presto/execution/DropFunctionTask.java

```diff
@@ -74,7 +74,14 @@ public String explain(DropFunction statement, List<Expression> parameters)
     }
 
     @Override
-    public ListenableFuture<?> execute(DropFunction statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, Session session, List<Expression> parameters, WarningCollector warningCollector)
+    public ListenableFuture<?> execute(
```

Comment:
unintentional change?

Reply:
not unintentional, I was making it more readable. Though unrelated, happy to move to separate PR if you want.